Packs/AWS-S3/Integrations/AWS-S3/AWS-S3.yml

```diff
@@ -505,6 +505,32 @@ script:
       the one in this request completely replaces it.
     execution: false
     name: aws-s3-put-bucket-policy
+  - arguments:
+    - description: The name of the bucket from which the server-side encryption configuration is retrieved.
+      name: bucket
+      required: true
+    - description: The account ID of the exepcted bucket owner.
+      name: expectedBucketOwner
+    - description: The AWS Region, if not specified the default region will be used.
+      name: region
+    - description: The Amazon Resource Name (ARN) of the role to assume.
+      name: roleArn
+    - description: An identifier for the assumed role session.
+      name: roleSessionName
+    - description: The duration, in seconds, of the role session. The value can range from 900 seconds (15 minutes) up to the maximum session duration setting for the role.
+      name: roleSessionDuration
+    description: Get AWS S3 Bucket Encryption
+    name: aws-s3-get-bucket-encryption
+    outputs:
+    - contextPath: AWS.S3.Buckets.BucketName.ServerSideEncryptionConfiguration.Rules.ApplyServerSideEncryptionByDefault.SSEAlgorithm
+      description: S3 Bucket Encryption SSE Algorithm.
+      type: string
+    - contextPath: AWS.S3.Buckets.BucketName.ServerSideEncryptionConfiguration.Rules.ApplyServerSideEncryptionByDefault.KMSMasterKeyID
+      description: S3 Bucket Encryption KMS Master Key ID.
+      type: string
+    - contextPath: AWS.S3.Buckets.BucketName.ServerSideEncryptionConfiguration.Rules.BucketKeyEnabled
```

Comment:
@YuvHayun I can't tell if I'm setting the `contextPath` correctly. Do I need that `BucketName` property in there? The `aws-s3-get-public-access-block` command includes the `BucketName` in the path, but the rest of the commands do not.

Reply:
Yes, keep the bucket name please.